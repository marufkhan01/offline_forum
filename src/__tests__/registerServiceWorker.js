import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import registerServiceWorker from './registerServiceWorker';



it('CreateNewPost component should render as expected',() =>{
    const component = shallow(<registerServiceWorker />);
    const tree = toJSON(component);
    expect(tree).toMatchSnapshot();
});