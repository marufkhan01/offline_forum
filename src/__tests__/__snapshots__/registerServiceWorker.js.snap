// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateNewPost component should render as expected 1`] = `<registerServiceWorker />`;
